core/mutation/mutation.go

```diff
@@ -76,7 +77,15 @@ func (s *Server) GetMutations(ctx context.Context, in *tpb.GetMutationsRequest)
 	}
 
 	// Get highest and lowest sequence number.
-	highestSeq := uint64(resp.GetMapRoot().GetMetadata().GetHighestFullyCompletedSeq())
+	meta, err := internal.MetadataFromMapRoot(resp.GetMapRoot())
+	if err != nil {
+		return nil, grpc.Errorf(codes.Internal, err.Error())
+	}
+	if meta.GetHighestFullyCompletedSeq() == 0 {
```

Comment:
Ah this is a bit of my dev logging that snuck in.  I can remove it if you want.

Reply:
yes please

Reply:
Done.